core/src/test/java/org/dozer/util/ResourceLoaderTest.java

```diff
@@ -38,13 +39,14 @@ public void testResourceNotFound() throws Exception {
 
   @Test
   public void testGetResourceWithWhitespace() {
-    URL url = loader.getResource(" contextMapping.xml ");
+    URL url = loader.getResource("mappings/contextMapping.xml ");
     assertNotNull("URL should not be null", url);
   }
 
+  @Ignore
```

Comment:
Would it be possible to re-enable this test so that after pull-request we do not have new ignored tests?